sound/soc/sof/loader.c

```diff
@@ -711,7 +711,7 @@ int snd_sof_load_firmware_raw(struct snd_sof_dev *sdev)
 	if (!fw_filename)
 		return -ENOMEM;
 
-	ret = request_firmware(&plat_data->fw, fw_filename, sdev->dev);
+	ret = firmware_request_nowarn(&plat_data->fw, fw_filename, sdev->dev);
```

Comment:
request_firmware() prints with dev_warn(), we print with dev_err() and we like to emphasise it with an added error: prefix.
Adding that error: to the already too long second line from us is too much and I could not find a better and shorter version for it.

Reply:
@lyakh, I have kept the original patch, but I don't have issue reworking it if you think it would be better.

Reply:
personally I usually prefer "standard" over "ad-hoc" so I'd rather have the standard error message instead of suppressing it and re-issuing effectively the same. But that's just my opinion, and it isn't hugely important anyway

Reply:
@lyakh, me too, I was not sure if SOF "standard" is preferred over "standard-standard", I'll push an update for this in couple of minutes.